### PR TITLE
User: view of the dishes on the menu page

### PR DESCRIPTION
**Pull request by Andycutter**
### PT link
https://www.pivotaltracker.com/story/show/158941644

### User story
```
As a User,
In order to see what is available for order
I would like to see dishes on the menu page
```

### What we learned
We learned to add form.number_field_tag
We learned there is a difference between the localhost-server and the "show me the page"-server
We learned to remove an unnecessary controller

### New additions
launchy gem added

### Wireframe
https://drive.google.com/drive/u/0/folders/15lUCqu1cJDHtWt_cHDF9Xu817KL05epo?ogsrc=32

### CODE REVIEW

**Review by oliverochman — APPROVED** [review]
LGTM!

File: features/display_dishes.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Feature: Display dishes on the menu page
+    As a User,
+    In order to decide what to eat,
+    I would like to see dishes on the menu page
+
+    Background: Dish
+        Given the following dishes exists
+        | name  | price | description   |
+        | Pasta | 100   | Very nice     |   
+        When I visit the menu page
```
Comment (faraznaeem) [inline features/display_dishes.feature]: Should be `And`

File: features/display_dishes.feature [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    Scenario: Display dishes on the menu page
+        Then I should see 'Pasta' in the dish table
```
Comment (faraznaeem) [inline features/display_dishes.feature]: Go over the scenario and background and feature and make sure they follow the right structure, `given when then`

File: app/views/menus/index.html.haml [hunk 3]
```diff
@@ -4,3 +4,32 @@
...
+        %td SEK
+
+        %td
+            = number_field_tag 'Quantity', nil, min: 1, max: 10, step: 1
+%p 
+    = link_to 'Order'
+
+
+
+
```
Comment (faraznaeem) [inline app/views/menus/index.html.haml]: Needless space

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**